test_controllingChargebyteBoard.py

```diff
@@ -0,0 +1,68 @@
+from controllingChargebyteBoard import *
+
+
+class Test:
+
+    def test_device_one(self):
```

Comment:
for all the other functions you can test mocking `send_packet` checking the service, payload and returning a constant response.
Make sure to also test error cases (e.g. no data returned)